_projects/2022/100348375/100348375.Rmd

```diff
@@ -0,0 +1,981 @@
+                     limits = c(10, 100),
+                     labels = c("10%", "20", "30", "40", "50", 
+                                "60", "70", "80", "90", "100"),
+                     name = NULL,
+                     expand = c(0, 0)) +
+  scale_y_discrete(name = NULL, expand = expansion(add = c(0.2, 0.8))) +
+  labs(subtitle = "Technology companies since 2012",
+       caption = 'By The New York Times | Source: IbisWorld and Open Markets Institute; "Early 2000s" ranges\n from 2002 to 2007, depending on data availability') +
+  theme(legend.position = "none",
+        plot.title.position = "plot",
+        plot.subtitle = element_text(size = 9.5, family = "Noto Serif", 
+                                     face = "bold", margin = margin(0, 0, 0, 0)),
+        plot.caption = element_text(size = 9, colour = "#636363", 
+                                    family = "Noto Serif", 
+                                    hjust = 0, margin = margin(20, 0, 0, 0)),
+        plot.caption.position = "plot",
+        panel.grid.major.y = element_line(linetype = "dotted", 
+                                          colour = "#a7a7a7"),
+        panel.grid.major.x = element_line(colour = "#c9c9c9"),
+        panel.grid.minor = element_blank(),
+        axis.ticks.y = element_blank(),
+        axis.ticks.x = element_line(colour = "#c9c9c9", 
+                                    size = c(0.3, 0.5, 0.5, 0.5, 0.8, 
+                                             0.5, 0.5, 0.5, 0.5, 0.3)),
+        axis.ticks.length.x = unit(0.2, "cm"),
+        axis.text.y = element_text(size = 10, family = "sans", 
+                                   hjust = 0, vjust = 0.5),
+        axis.text.x = element_text(size = 9, family = "sans", 
+                                   colour = "#a7a7a7",
+                                   face = c('plain', 'plain', 'plain', 
+                                            'plain', 'bold',
+                                            'plain', 'plain', 'plain', 
+                                            'plain', 'plain')),
+        panel.background = element_rect(fill = "white"),
+        plot.margin = margin(0, 1.7, 0, 0, "cm")) +
+  # annotate("text", x = 75, y = 3.5, label = "2012", size = 3, fontface = "bold") +
+  # annotate("text", x = 97, y = 3.5, label = "Now", size = 3, fontface = "bold") +
+  annotation_custom(grob = textGrob(label = "Now", 
+                                    gp = gpar(fontsize = 8.5, fontface = "bold", 
+                                              fontfamily = "sans")),
+                    ymin = 3, xmin = 94.3) +
+  annotation_custom(grob = textGrob(label = "2012", 
+                                    gp = gpar(fontsize = 8.5, fontface = "bold", 
+                                              fontfamily = "sans")),
+                    ymin = 3, xmin = 52) # También tiene como límite los márgenes del área del gráfico.
+
+bottom_plot
+
+```
+<br/><br/>
```

Comment:
Please remove all manual line breaks like this one from the post. Let the CSS lay out the post. :)